src/redux/categories/categories.js

```diff
@@ -0,0 +1,18 @@
+const CHECKING_STATUS = 'categories/CHECKING_STATUS';
```

Comment:
- [ ] Kindly stick with the name convention of the [duck pattern](https://github.com/erikras/ducks-modular-redux) for naming actions also here.